src/uucore/src/lib/features/fs.rs

```diff
@@ -798,6 +800,20 @@ pub fn get_filename(file: &Path) -> Option<&str> {
     file.file_name().and_then(|filename| filename.to_str())
 }
 
+/// Make a FIFO, also known as a named pipe.
+///
+/// This is a safe wrapper for the `mkfifo` function from `libc`.
+#[cfg(unix)]
+pub fn make_fifo(path: &Path) -> std::io::Result<()> {
```

Comment:
given that the doc will be published here:
https://docs.rs/uucore/latest/uucore/fs/index.html

please add some rustdoc (with an example :)
thanks

Reply:
also, please add an unit test